scripts/build.sh

```diff
@@ -16,12 +18,25 @@ case "$RUNNER_OS" in
     EXECUTABLE_EXT=".exe"
 esac
 
+case "$ARCH" in
+  arm64)
+    ARCH_OPT="--arm64=true"
+    ;;
+  *)
+    ARCH_OPT=""
+    ;;
+esac
+
 pushd repos/$SOLVER
 if [[ "$RUNNER_OS" == 'Windows' ]] ; then
   sed -i.bak -e 's/STATIC_BIN=False/STATIC_BIN=True/' scripts/mk_util.py
 fi
-python scripts/mk_make.py
+python scripts/mk_make.py $ARCH_OPT
 (cd build && make -j4 && cp z3$EXECUTABLE_EXT $BIN/$SOLVER$EXECUTABLE_EXT)
 strip $BIN/$SOLVER$EXECUTABLE_EXT
-$BIN/$SOLVER$EXECUTABLE_EXT --version
+
+if [[ "$ARCH" == 'x64' ]] ; then
```

Comment:
Is there a reason this doesn't work on `arm64`?

Reply:
`arm64` binaries are produced by cross compilation on an `x64` github machine, so we can't just run and test them without using some kind of emulation. Any attempt to run it will fail with the `Bad CPU type in executable` error.

Reply:
In the future we can try to replace cross compilation with native `arm64` builds using https://github.com/github/roadmap/issues/528